crates/meteora-parser/src/custom_impl/oracle.rs

```diff
@@ -0,0 +1,34 @@
+use borsh::BorshDeserialize;
```

Comment:
What Ive been doing is adding extra to the lib.rs that way the generated mod is always codama generated but can add more to the lib.rs.